cmd/submit.go

```diff
@@ -141,6 +141,11 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 		return err
 	}
 
+	if exercise.Slug != metadata.Exercise {
+		// TODO: error msg should suggest running future doctor command
+		return errors.New("exercise directory does not match exercise slug in metadata")
```

Comment:
Maybe we should indicate what was found, what was expected, and how to fix. When users open issues pertaining to path errors the messaging doesn't contain path information, which makes the troubleshooting a little harder than it needs to be. 

`exercise directory does not match slug: expected %s but found %s. Please rename the directory and try again. `